.bit/responses/Format.md

```diff
@@ -0,0 +1,35 @@
+---
+files: n/a
+stepType: checks
+scripts: test.parseimage.js
+week: 2
+step: 1
+name: Week 2 Step 1
+---
+
+**Week 1 Step 1** ⬤◯◯
+
+🕐 **Estimated completion:** 25-35mins
+
+## Learning GitHub
+### ✅ Task:
+
+**TASK DESCRIPTION.**
+- [ ] 
+
+## ❓ If you are confused
+
+<details>
+<summary>❓ </summary>
+
+</details>
+
+<details>
+<summary>❓ </summary>
+
```

Comment:
One big problem everyone (including myself) had trouble with is that understanding the markdown inside dropdowns wouldn't work unless you put a space under the summary tags